app/services/save_practice_service.rb

```diff
@@ -36,23 +36,25 @@ def save_practice
       if @practice_params["practice_multimedia_attributes"].present?
         process_multimedia_params
       end
+      if @practice_params["practice_resources_attributes"].present?
+        process_practice_resources_params
+      end
       if @practice_params["risk_mitigations_attributes"].present?
         process_risk_mitigations_params
       end
 
-        updated = @practice.update(@practice_params)
-        rescue_method(:update_practice_partner_practices)
-        rescue_method(:update_department_practices)
-        rescue_method(:remove_attachments)
-        rescue_method(:manipulate_avatars)
-        rescue_method(:remove_main_display_image)
-        rescue_method(:crop_main_display_image)
-        rescue_method(:update_initiating_facility)
-        rescue_method(:update_practice_awards)
-        rescue_method(:update_category_practices)
-        rescue_method(:crop_resource_images)
-
-        updated
+    @practice.update(@practice_params)
+    rescue_method(:update_practice_partner_practices)
+    rescue_method(:update_department_practices)
+    rescue_method(:remove_attachments)
+    rescue_method(:manipulate_avatars)
+    rescue_method(:remove_main_display_image)
+    rescue_method(:crop_main_display_image)
+    rescue_method(:update_initiating_facility)
+    rescue_method(:update_practice_awards)
+    rescue_method(:update_category_practices)
+    rescue_method(:crop_resource_images)
+      true
```

Comment:
should we have this returning `updated` as it was before?

Reply:
yeah - set this back to updated  - result of calling the save_practice method.